keda/mysql-hpa.yaml

```diff
@@ -20,4 +20,5 @@ spec:
         dbName: "mysql"
         port: "3306"
         queryValue: "5"
-        query: "SELECT CEIL(COUNT(*) / 10) FROM test WHERE state='QUEUED'"
\ No newline at end of file
+        query: "SELECT CEIL(COUNT(*) / 10) FROM test WHERE state='QUEUED'"
+        
```

Comment:
```suggestion

```
No white spaces. Otherwise Github still will repot no new line
<img width="886" alt="Screenshot 2021-03-04 at 10 50 01" src="https://user-images.githubusercontent.com/9528307/109945312-62109980-7cd7-11eb-80ad-5fe43c2e2a79.png">